apps/web/src/api/schedules.ts

```diff
@@ -0,0 +1,16 @@
+import { SERVER_URL } from '@/config'
+import { IMakeNewScheduleRequest } from '@fienmee/types'
+
+export async function makeNewSchedule(request: IMakeNewScheduleRequest) {
+    const res = await fetch(`${SERVER_URL}/v1/schedules`, {
+        method: 'POST',
+        authorization: `bearer ${token}`,
```

Comment:
```suggestion
        headers: { Authorization: `bearer ${token}` },
```

token 변수가 없어서 build 시에 여기서도 오류 발생하겠네요